docs/src/docs/modules/layout-check.adoc

```diff
@@ -12,15 +12,17 @@ Although, manual inspection can never be pixel-accurate. An inspection of perhap
 
 At this point an automatic comparison of reference screenshots can help. This test is to be seen in addition to the functional tests, but cannot replace them. In a test case, it would also be conceivable to combine the check of function and layout.
 
-== Description
+== How it works
 
 A layout test with the Testerra utilities is actually a comparison between a reference and the actual state. This is done via the screenshot functionality of Selenium, in which both screenshots (reference and actual) are compared pixel by pixel from top to bottom. In this comparison, a third image is created in which differences (ReferencePixel != ActualPixel) are marked in red.
 
+In case that one image is a different size than the other an <<#_optional_assertions, Optional assertion>> fails, so that an error with the information about the different sizes is displayed in the report but the Layout test will still be fully executed. Testerra then considers only the common pixels of both images as 100%. Pixels that are outside of one or the other image are not included in the error calculation. In the comparison, those Pixels are marked in blue.
```

Comment:
Typo: ... that one image [has] a different size...

Reply:
I suggest to shorten that chapter:
... than the other [Testerra reports this as an <<#_optional_assertions, optional assertion>>. Testerra considers only the common]...